dotcom-rendering/cypress.config.js

```diff
@@ -1,11 +1,14 @@
 import path from 'node:path';
+import { fileURLToPath } from 'node:url';
```

Comment:
I'm not sure if we're using this?

Reply:
Follow-up PR: try to enforce linting and running TS against this file, see https://github.com/guardian/dotcom-rendering/pull/8482/files#diff-ed2946c80569ace27e3818925bb9fbcd193e0292796204b0390f21435b0929a0R22-R24

Good luck!